mbuild/tests/test_lib_molecule.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+
+from mbuild.lib.molecules import (
+    WaterSPC,
+    WaterTIP3P,
+    WaterTIP4P,
+    WaterTIP4P2005,
+    WaterTIP4PIce,
+)
+from mbuild.tests.base_test import BaseTest
+
+
+class TestWater(BaseTest):
+    def test_tip3p(self):
```

Comment:
If you want to reduce the tests, I think you could do something like this:
```
    @pytest.mark.parametrize("model, bl, angle", [(WaterTIP3P, 0.09572, 104.52)])
    def test_3site(self, model, angle):
        water = model()
```

Reply:
Great suggestion @rmatsum836. Once again, laziness had gotten the best of me :).